src/qt/qvalidatedlineedit.cpp

```diff
@@ -28,7 +34,7 @@ void QValidatedLineEdit::setValid(bool _valid)
     }
     else
     {
-        setStyleSheet(STYLE_INVALID);
+        setStyleSheet("QValidatedLineEdit { " STYLE_INVALID "}");
```

Comment:
Are quotes around `STYLE_INVALID` macro really needed?

Reply:
What?

Reply:
Oh, misreading. Sorry for noise.